src/main/java/qna/domain/Answer.java

```diff
@@ -82,47 +95,54 @@ public Question question() {
 
 	public void question(Question question) {
 		this.question = question;
-	}
-
-	public String contents() {
-		return contents;
+		updatedAtNow();
 	}
 
 	public boolean isOwner(User writer) {
 		return this.writer.equals(writer);
 	}
 
+	public void validateIsOwner(User loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
	private void validateIsOwner(User loginUser) {
```

Answer 객체의 유효성 검사를 외부에서 하는 것 보다 내부에서 상태 바꾸기 전에 검사하도록 만들면 어떨까요
클라이언트에서 검사하도록 만들면 유효성 검사를 누락할 수 있습니다.

Reply:
아.. 드디어 이해했습니다! 💡 💡 💡 💡 💡 
리뷰어님께서 접근제어자에 대해 이렇게 꼼꼼하게 관찰 👀 하시고 계시다는 것에 
감탄 ❗ 하고 있습니다!!!!
그리고 제가 얼마나 무심한 코드 👨‍💻 를 작성하고 있었는지 되돌아보게 되었습니다. 😞 
감사합니다!! 🙇